### PR TITLE
Improve pipeline model text description clarity

### PR DESCRIPTION
**Pull request by wolenetz**
Fixes a typo (one of the flows was described incorrectly as going to a
Video Decoder, not an Audio Decoder), and fixes a couple omissions of
mentions of track buffer process indications between SourceBuffer 2
and Video Decoder, and between SourceBuffer 3 and Audio Decoder.

(I found these issues while investigating #307 and reviewing #308.)

### CODE REVIEW

**Comment by wolenetz** [comment]
@mwatson2 - please review.

@tidoust - with #308 now merged to main, is there anything further needed to ensure both WD and ED have the updated text description once this PR is approved, squashed and merged to main?

**Comment by tidoust** [comment]
> @tidoust - with https://github.com/w3c/media-source/pull/308 now merged to main, is there anything further needed to ensure both WD and ED have the updated text description once this PR is approved, squashed and merged to main?

Well, there seems to be a bug somewhere that crashes the update of the `gh-pages` branch :( I don't yet know what triggers it, perhaps an incompatibility between the part of the job that handles the static content and the part of the job that handles the spec itself. I will look into it. No need to block any of the pending pull requests on this, I'll re-generate the `gh-pages` branch once I have found and fixed the bug!

**Comment by wolenetz** [comment]
>  I will look into it. No need to block any of the pending pull requests on this, I'll re-generate the gh-pages branch once I have found and fixed the bug!

Thank you. I'll await @mwatson2 review of this and eventually `gh-pages` will be right again :)

**Comment by wolenetz** [comment]
@mwatson2 - please review this when you can. To clear the editorial PR backlog, I'm going to squash and merge this today though since it is indeed editorial.

File: pipeline_model_description.html [hunk 1]
```diff
@@ -46,29 +45,35 @@ <h3 id='sourcebuffer1'>Flowing down from SourceBuffer 1</h3>
...
 
 <p>Flowing down from SourceBuffer 3 is a rose triangle with an arrow flowing to
-a green box labeled Audio Decoder, which then flows through a closed switch,
-which then also continues to the same indicator (green circle with an X)
-labeled Audio Device previously mentioned in SourceBuffer 1</p>
+a process indication labeled Track Buffer.<p>
+
+<p>This Track Buffer then flows to a green box labeled Audio Decoder, which
+then flows through a closed switch, which then also continues to the same
+indicator (green circle with an X) labeled Audio Device previously mentioned in
```
Comment (chrisn) [inline pipeline_model_description.html:75]: I think we could rephrase this to describe the purpose of "indicator (green circle with an X)". In a flow chart (I just looked this up...) the symbol is a "summing junction" which suggests that any enabled audio tracks are summed, i.e., mixed to form the output audio. This corresponds with definition of `AudioTrack`. 

Unless you prefer to do it, I could raise a PR with suggested text along those lines.

Comment (wolenetz) [reply pipeline_model_description.html:75]: Good catch. PR sounds good to me if you have time. I've filed an issue to ensure this gets done one way or the other :)
https://github.com/w3c/media-source/issues/314
